clients/erlang/src/simple_space_ai.erl

```diff
@@ -0,0 +1,46 @@
+-module(simple_space_ai).
+
+-export([give_moves/5]).
+
+-include("include/space.hrl").
+
+give_moves(RoundId,
+           #config{bots         = _ConfigBots,
+                   field_radius = _ConfigFieldRadius,
+                   move         = _ConfigMove,
+                   start_hp     = _ConfigStartHp,
+                   cannon       = _ConfigCannon,
+                   radar        = _ConfigRadar,
+                   see          = _ConfigSee,
+                   max_count    = _ConfigMaxCound,
+                   asteroids    = _ConfigAsteroids,
+                   loop_time    = _ConfigLoopTime,
+                   no_wait      = _ConfigNoWait} = _Config,
+           #team{name    = _MyTeamName,
+                 team_id = _MyTeamId,
+                 bots    = MyBots} = _MyTeam,
+           _OtherTeams,
+           _Events) ->
+
+    %% Look at current state through ETS and make a decision
+    [{team_id, SavedTeamId}] = ets:lookup(space_tyckiting, team_id),
+    io:format("My saved team id: ~p~n", [SavedTeamId]),
+
+    %% Save something for next round:
+    ets:insert(space_tyckiting, {my_memo, something}),
```

Comment:
Not sure what would be more erlangy, but sure it would be possible to avoid using ets in the skeleton client and instead use the "gen_server" like state passing that you suggest. Will fix